app/services/api/latest_additions.rb

```diff
@@ -0,0 +1,40 @@
+module Api
+  class LatestAdditions
+    def initialize(count)
+      @count = count
+    end
+
+    def call
+      (serialize_litigations + serialize_legislations).sort_by { |item| - item[:created_at].to_time.to_i }
```

Comment:
it's just a question, maybe it's supposed to be this way... But I thought that the latest additions don't mean latest in the terms of adding them to the DB (`created_at`) but more in a sense of `date_passed` in Legislation

Reply:
Yes, we don't have the right data in, so I'd leave as this for now and then adjust once we got some importers in.